src/js/edu-benefits/5490/config/form.js

```diff
@@ -248,19 +249,34 @@ const formConfig = {
               },
               remarriageDate: _.assign(date.uiSchema('Date you got remarried'), {
                 'ui:options': {
-                  hideIf: (formData) => !_.get('spouseInfo.remarried', formData)
-                }
+                  expandUnder: 'remarried',
+                },
+                'ui:required': (formData) => _.get('spouseInfo.remarried', formData)
               }),
               'ui:options': {
                 hideIf: (formData) => _.get('relationship', formData) !== 'spouse'
               }
             },
-            relativeFullName: _.assign(fullNameUISchema, {
-              'ui:title': 'Name of Sponsor'
+            veteranFullName: _.merge(fullNameUISchema, {
+              'ui:title': 'Name of Sponsor',
+              first: {
+                'ui:title': 'Veteran first name'
```

Comment:
May want to relabel these using "Sponsor" per https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1696

I think to avoid confusion if the applicant is also a veteran. "Wait, which veteran? My sponsor or me?"